### PR TITLE
refactor beego tests

### PR DESCRIPTION
**Pull request by sergeylanzman**
1. Refactor - create app like beego app.
2. Add queries test

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
Hi @sergeylanzman ! Thanks for the contribution. Before applying changes to any frameworks going forward, one thing we are requiring is that packages/vendors/git repos be tied to a specific version or commit. This helps limit the number of breaking tests in future rounds. Would it be possible for you to use [glide](https://github.com/Masterminds/glide) here? You can also see the Echo framework's implementation [here](https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Go/echo/src).

**Comment by sergeylanzman** [comment]
OK I will fixed this later.

File: frameworks/Go/beego/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is the Beego portion of a [benchmarking test suite](../) comparing a variet
 
 ## Versions
 
-* [Go 1.1](http://golang.org/)
+* [Go 1.8](http://golang.org/)
```
Comment (methane) [inline frameworks/Go/beego/README.md]: https

File: frameworks/Go/beego/benchmark_config.json [hunk 2]
```diff
@@ -18,7 +19,7 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "beego",
+      "display_name": "beego-mysql-orm",
```
Comment (methane) [inline frameworks/Go/beego/benchmark_config.json]: "beego" is enough, until other flavor (postgres, mysql-raw) implemented.

File: frameworks/Go/beego/setup.sh [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
 
-sed -i 's|tcp(.*:3306)|tcp('"${DBHOST}"':3306)|g' src/hello/hello.go
+sed -i 's|tcp(.*:3306)|tcp('"${DBHOST}"':3306)|g' src/hello/models/init.go
 
 fw_depends mysql go
 
-go get github.com/astaxie/beego
-go get github.com/go-sql-driver/mysql
+go get -u github.com/astaxie/beego
+go get -u github.com/go-sql-driver/mysql
```
Comment (methane) [inline frameworks/Go/beego/setup.sh]: Could you use glide here too?

**Comment by sergeylanzman** [comment]
@nbrady-techempower  @methane done

**Comment by nbrady-techempower** [comment]
@sergeylanzman Round 14 preview is already out. Preview 2 should be out around Monday. If you're adding more tests, this won't make Round 14.

**Comment by sergeylanzman** [comment]
@nbrady-techempower  Now this done.

**Comment by nbrady-techempower** [comment]
Ok im going to merge this in because it includes version locking that I would like associated with the R14 release. Thanks @sergeylanzman & @methane for your review!

**Pull request merged**